test/integration.test.js

```diff
@@ -37,16 +45,11 @@ describe('registry-data (integration)', function () {
 
 
   before(function (done) {
-    if (process.env.DEBUG) { // eslint-disable-line no-process-env
-      datastar.connection.on('queryStarted', function () {
-        console.log.apply(console, arguments);
-      });
-    }
-    datastar.connect(done);
-  });
-
-  after(function (done) {
-    datastar.close(done);
+    liveness.waitForServices({
+      clients: [dynamoDriver],
+      waitSeconds: 60
+    }).then(() => done())
+      .catch(done);
```

Comment:
the before/after mocha functions can be async as well, so you could use `await` here if you wanted. otherwise this is fine.